src/modules/browser/speech-synthesis/store-module-browser.tsx

```diff
@@ -0,0 +1,40 @@
+import { create } from 'zustand';
+import { persist } from 'zustand/middleware';
+import { useShallow } from 'zustand/react/shallow';
+
+export type BrowsePageTransform = 'html' | 'text' | 'markdown';
+
+interface BrowseState {
+
```

Comment:
I'm wondering if the settings also of TTSEngine could be here, to keep everything together.